addons/payment/models/account_move.py

```diff
@@ -13,6 +13,8 @@ class AccountMove(models.Model):
     authorized_transaction_ids = fields.Many2many(
         string="Authorized Transactions", comodel_name='payment.transaction',
         compute='_compute_authorized_transaction_ids', readonly=True, copy=False)
+    total_amount_tx = fields.Monetary(string='Amount paid', store=True,
+        compute='_compute_total_amount_tx')
```

Comment:
Why do you store this field?

Reply:
No idea. Seems to works perfectly fine without storing it so I removed the tag.